olad/www/mobile.html

```diff
@@ -448,6 +448,15 @@
       <div id="plugin_info_frame"></div>
     </div>
 
+    <noscript>
+      <br>
+      <p align="center">
+        <strong>This interface requires JavaScript to function. If you
+        would like to use it, enable JavaScript and reload the
```

Comment:
I thought about that, and almost wrote it that way, but decided that it was the user’s choice. If you like I’d be happy to add it while I am in there fixing the indentations.